TFVC_README.md

```diff
@@ -0,0 +1,198 @@
+
+Here are the currently supported features provided by the extension:
+- Execute all basic version control actions such as add, delete, rename, move, etc.
+- View local changes and history for your files
+- Include and Exclude changes (and move files between the two states)
+- Merge conflicts from updates
+- Check-in and update local files
+- Associate work items to check-ins
+- Integrated TFVC Output window
+- Support for a TFS proxy
+- Support for workspaces created with either the Visual Studio IDE, the JetBrains IDEs and Eclipse
+
+![Team Foundation Version Control Viewlet](assets/tfvc-viewlet.png)
+
+## Getting Started Videos
+Below are a few videos to help get you started using TFVC quickly.
+- [TFVC Source Code Control for Visual Studio Code](https://youtu.be/6IzJ2UPGmoQ) - This video shows you how to set up
+the TFVC support on Windows and demonstrates much of the functionality available for Team Foundation Version Control. The
+features shown in this video apply equally well to the TFVC support on macOS and Linux.
+- [Set up and Configure the TEE CLC on Linux (and macOS)](https://youtu.be/VPNaEIVZfr0) - This video demonstrates how to set
+up the Team Explorer Everywhere Command Line Client (TEE CLC) on Ubuntu. On macOS and Linux, the CLC provides the TFVC capability
+to the extension.
+- [Set up the Team Services extension for Visual Studio Code](https://youtu.be/t6gGfj8WOgg) - If you haven't used the extension
+before, this video will show you how to set it up, create a personal access token and get up and running.
+- [Walkthrough of the Team Services extension for Visual Studio Code](https://youtu.be/sk6LrzQX4P8) - This is a walkthrough of
+most of the features of the Team Services extension.
+
+## Prerequisites
+### Visual Studio Team Services
+If you are planning on using the extension with Visual Studio Team Services, ensure you have a Team Services account. If you do
+not have an account, find out how to [sign up for Team Services](https://www.visualstudio.com/en-us/get-started/setup/sign-up-for-visual-studio-team-services).
+
+### Team Foundation Server
+If you are planning on using the extension with Team Foundation Server, you **must** be running Team Foundation
+Server 2015 Update 2 or later. Earlier versions of Team Foundation Server are not supported.
+
+## Installation
+First, you will need to install [Visual Studio Code](https://code.visualstudio.com/download) `1.11.1` or later.
+
+### The Visual Studio Code Extension
+To install the extension with the latest version of Visual Studio Code (version 1.11.1 is the latest as of this writing), bring
+up the Visual Studio Code Command Palette (`F1`), type `install` and choose `Extensions: Install Extensions`. In the `Search Extensions in Marketplace`
+text box, type `team`. Find the `Visual Studio Team Services` extension published by *Microsoft* and click the `Install` button.
+Restart Visual Studio Code.
+
+### TFVC Command Line Client
```

Comment:
You don't mention the actual tfvc.location setting that they need to set in this section. Seems like it might make sense here.